### PR TITLE
Add schemas/dev.amp.snowplow/amp_consent/jsonschema/1-0-0

### LINKED ISSUE #1328: Add dev.amp.snowplow/amp_consent/jsonschema/1-0-0

### PR DESCRIPTION
**Pull request by igneel64**
Add AMP consent schema for the amp tracker feature https://github.com/snowplow-incubator/amphtml/pull/19

close #1328

### CODE REVIEW

**Review by matus-tomlein — APPROVED** [review]
LGTM!

Based on the implementation, it seems like the properties could also be required, but no problem if you think that it's safer to keep them optional.

File: schemas/dev.amp.snowplow/amp_consent/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "maxLength": 128
+        },
+        "gdprApplies": {
+            "type": ["boolean", "null"],
+            "description": "If GDPR applies on the current consent policy. Part of the consent metadata."
+        },
+        "consentType": {
+            "type": ["string", "null"],
+            "description": "Type of consent given as an enum. At the time 1: TCF_V1, 2: TCF_V2, US_PRIVACY_STRING: 3 Part of the consent metadata.",
+            "maxLength": 32
```
Comment (colmsnowplow) [inline schemas/dev.amp.snowplow/amp_consent/jsonschema/1-0-0:24]: Is it possible for the enum to change in future, and a value longer than 32 bits be used?

If we think that's likely we might consider a max length which predicts it, but nothing wrong with 32!

**Review by colmsnowplow — APPROVED** [review]
LGTM!

File: schemas/dev.amp.snowplow/amp_consent/jsonschema/1-0-0 (hunk 1 above)
Comment (igneel64) [reply schemas/dev.amp.snowplow/amp_consent/jsonschema/1-0-0:24]: Pushing it to 64 and adding to release! Thanks!

**Comment by adatzer** [comment]
Rebased and merged into `release/r148`. Closing.

**Pull request closed without merging**